.buildkite/src/Monorepo.dhall

```diff
@@ -32,10 +30,11 @@ in Pipeline.build Pipeline.Config::{
   },
   steps = [
     Command.Config::{
-      command = [ prepareCommand ] # commands,
+      command = commands,
       label = "Monorepo triage",
       key = "cmds",
-      target = Size.Small
+      target = Size.Small,
+      docker = Docker.Config::{ image = "codaprotocol/ci-toolchain-base" }
```

Comment:
This should probably have a version tag so that it doesn't just pull latest (which is bad since we may have different branches with different image versions for their CI). Also, can you configure this value as a constant coming from some file? For instance, `(./Constants/ContainerImages.dhall).toolchainBase`. That way it's easy to update it later.